teuthology/suite/run.py

```diff
@@ -186,8 +196,11 @@ def choose_ceph_hash(self):
             log.info("ceph sha1 explicitly supplied")
 
         elif self.args.ceph_branch:
-            ceph_hash = util.git_ls_remote(
+            try:
+                ceph_hash = util.git_ls_remote(
                 self.args.ceph_repo, self.args.ceph_branch)
```

Comment:
```suggestion
                    self.args.ceph_repo, self.args.ceph_branch)
```

Reply:
thank you, resolved with the new commit.